instrumentation/alibaba-druid-1.0/library/src/test/java/io/opentelemetry/instrumentation/alibaba/v1_0/DruidInstrumentationTest.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.alibaba.v1_0;
```

Comment:
I think it should be `io.opentelemetry.instrumentation.alibabadruid.v1_0`

Reply:
@laurit fixed, thank you